modules/core/client/views/groupinfo.client.view.html

```diff
@@ -1,55 +1,111 @@
+                    <tr><th><button class="btn" style="background:transparent">1895</button></th><th>95%</th></tr>
+                    <tr><th><button class="btn" style="background:transparent">4597</button></th><th>95%</th></tr>
+                    <tr><th><button class="btn" style="background:transparent">3541</button></th><th>95%</th></tr>
+                    <tr><th><button class="btn" style="background:transparent">6584</button></th><th>95%</th></tr>
+                    <tr><th><button class="btn" style="background:transparent">1568</button></th><th>95%</th></tr>
+                    <tr><th><button class="btn" style="background:transparent">4959</button></th><th>95%</th></tr>
+                    <tr><th><button class="btn" style="background:transparent">7562</button></th><th>95%</th></tr>
+                    <tr><th><button class="btn" style="background:transparent">1621</button></th><th>95%</th></tr>
+                    <tr><th><button class="btn" style="background:transparent">1624</button></th><th>95%</th></tr>
+                    <tr><th><button class="btn" style="background:transparent">6875</button></th><th>95%</th></tr>
+                    <tr><th><button class="btn" style="background:transparent">1895</button></th><th>95%</th></tr>
+                    <tr><th><button class="btn" style="background:transparent">4597</button></th><th>95%</th></tr>
+                    <tr><th><button class="btn" style="background:transparent">3541</button></th><th>95%</th></tr>
+                    <tr><th><button class="btn" style="background:transparent">6584</button></th><th>95%</th></tr>
+                    <tr><th><button class="btn" style="background:transparent">1568</button></th><th>95%</th></tr>
+                    <tr><th><button class="btn" style="background:transparent">4959</button></th><th>95%</th></tr>
+                    <tr><th><button class="btn" style="background:transparent">7562</button></th><th>95%</th></tr>
+                    <tr><th><button class="btn" style="background:transparent">1621</button></th><th>95%</th></tr>
+                    <tr><th><button class="btn" style="background:transparent">1624</button></th><th>95%</th></tr>
+                    <tr><th><button class="btn" style="background:transparent">6875</button></th><th>95%</th></tr>
+                    <tr><th><button class="btn" style="background:transparent">1895</button></th><th>95%</th></tr>
+                    <tr><th><button class="btn" style="background:transparent">4597</button></th><th>95%</th></tr>
+                    <tr><th><button class="btn" style="background:transparent">3541</button></th><th>95%</th></tr>
+                    <tr><th><button class="btn" style="background:transparent">6584</button></th><th>95%</th></tr>
+                </tbody>
+            </table>
+        </div> 
+        <div class="col-xs-7 col-sm-7 col-md-7 col-lg-7 col-sm-offset-1 col-xs-offset-1 col-md-offset-1 col-lg -offset-1">
```

Comment:
!fix At the end of the line with the lg offset, there is a space between lg and the '-', this needs to be fixed or else I do not think it will work